server/controllers/auth/login.js

```diff
@@ -0,0 +1,68 @@
+const bcrypt = require('bcrypt');
+const { sign } = require('jsonwebtoken');
+
+const loginValidation = require('../validation/loginValidation');
+const Admin = require('../../database/models/admin');
+const Client = require('../../database/models/client');
+
+require('env2')('config.env');
+
+const login = async (req, res) => {
+  const { email, password, mobileNumber } = req.body;
+  try {
+    const valid = await loginValidation(email, mobileNumber, password);
+    if (!valid) {
+      res.status(400).json({
+        status: 'failed',
+        message: 'Check the entered data',
+      });
+    } else {
+      const clients = await Client.findOne({
+        $or: [{ email }, { mobileNumber }],
+      });
+      await bcrypt.compare(password, clients.password, (err, result) => {
```

Comment:
i prefer if you use [ try, catch ] here, it will catch any errors in unsynchronized function.